mender-convert-modify

```diff
@@ -277,6 +222,61 @@ ${data_part_device}   /data          auto       ${MENDER_DATA_PART_FSTAB_OPTS}
 $fstab
 EOF
 
+log_info "Installing Mender client and related files"
+
+if [ "${MENDER_CLIENT_INSTALL}" = "y" ]; then
+    case "${MENDER_CLIENT_VERSION}" in
+        1.* | 2.* | 3.*)
+            log_info "Installing Mender client version ${MENDER_CLIENT_VERSION}"
```

Comment:
I'm going to leave a few comments on pre-existing stuff, feel free to do with it as you like.

But this message should be outside the `case`. It's generic

Reply:
The message is different for the two cases due to the different packages being installed. I'd rather leave it like this.